pom.xml

```diff
@@ -22,1770 +22,41 @@
-    <version.apache.commons-beanutils>1.9.4</version.apache.commons-beanutils>
-    <version.apache.commons-pool>1.6</version.apache.commons-pool>
-    <version.apache.commons-dbcp>1.4</version.apache.commons-dbcp>
-    <version.apache.commons-digester>2.1</version.apache.commons-digester>
-    <version.apache.commons-logging>1.2</version.apache.commons-logging>
-    <version.twitter4j>3.0.5</version.twitter4j>
-    <version.restfb>1.6.12</version.restfb>
-    <version.google.oauth.client>1.33.3</version.google.oauth.client>
-    <version.google.apis.oauth2>v2-rev157-1.25.0</version.google.apis.oauth2>
-    <version.google.api.client>1.35.0</version.google.api.client>
-    <version.google.http.client>1.35.0</version.google.http.client>
-    <version.google.http.client.jackson>1.29.2</version.google.http.client.jackson>
-    <version.google.apis.plus>v1-rev590-1.25.0</version.google.apis.plus>
-    <version.jackson.core>2.13.2</version.jackson.core>
-    <version.codehaus.jackson.core.asl>1.9.13</version.codehaus.jackson.core.asl>
-    <version.reflext>1.1.0</version.reflext>
-    <version.scribejava>6.9.0</version.scribejava>
-    <org.lombok.version>1.18.24</org.lombok.version>
-    <org.gatein.api.version>1.0.1.Final</org.gatein.api.version>
-    <org.gatein.mop.version>1.3.2.Final</org.gatein.mop.version>
-    <org.gatein.mgmt.version>2.1.0.Final</org.gatein.mgmt.version>
-    <org.gatein.naming.version>1.1.1.Final</org.gatein.naming.version>
-    <org.gatein.common.version>2.2.2.Final</org.gatein.common.version>
-    <org.gatein.captcha.version>1.1.1.Final-gatein-4</org.gatein.captcha.version>
-    <!-- 3rd party deprecated libraries versions -->
-    <org.ow2.jotm.version>2.1.9</org.ow2.jotm.version>
-    <ow2-jta-1.1-spec.version>1.0.13</ow2-jta-1.1-spec.version>
-    <transactions-jta.version>3.8.0</transactions-jta.version>
-    <com.experlog.xapool.version>1.5.0</com.experlog.xapool.version>
-    <!-- For testing only -->
-    <version.arquillian>1.1.12.Final</version.arquillian>
-    <version.arquillian.tomcat>1.1.0.Final</version.arquillian.tomcat>
-    <version.shrinkwrap>1.2.6</version.shrinkwrap>
-    <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
-    <!-- Servers -->
-    <org.apache.tomcat.version>9.0.60</org.apache.tomcat.version>
-    <com.fasterxml.jackson.version>2.13.3</com.fasterxml.jackson.version>
-    <org.graalvm.js.version>22.0.0</org.graalvm.js.version>
   </properties>
   <dependencyManagement>
     <dependencies>
-      <!-- Logging implementation : http://logback.qos.ch/ -->
       <dependency>
-        <groupId>ch.qos.logback</groupId>
-        <artifactId>logback-classic</artifactId>
-        <version>${ch.qos.logback.version}</version>
+        <groupId>org.exoplatform.social</groupId>
+        <artifactId>social</artifactId>
+        <version>${org.exoplatform.social.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
       </dependency>
       <dependency>
```

Comment:
these deleted dependencies will be herited from meeds one ?

Reply:
No, when creating the module, ITOP copy the pom from maven-depmgt-pom as is.
As dependencies are already declared in maven-depmgt-pom, we dont need it here.

We didnt evaluate which dep should be in maven-exo-depmgt-pom or in maven-depmgt-pom, for the moment we only create the module structure.
When all is good for that, we will work on moving deps declaration